src/components/organisms/PostContent/PostContent.tsx

```diff
@@ -21,10 +23,12 @@ export function PostContent({ postId, className }: PostContentOrganismProps) {
   }
 
   return (
-    <div className={Libs.cn('flex flex-col', className)}>
+    <Atoms.Container className={Utils.cn('gap-3', className)}>
```

Comment:
Keep consistent with other components in the codebase

```suggestion
    <Atoms.Container className={Libs.cn('gap-3', className)}>
```

Reply:
I thought we talked about this on our last Franky call based on my comment in Slack:

<img width="1211" height="588" alt="image" src="https://github.com/user-attachments/assets/f8b12933-9002-4ed0-8825-a3db3832521e" />

We decided it would be better to not import the entire `Libs` if you only need to use a small subset of it in your file. Or are we not making that change until after the initial release?